docs/src/main/asciidoc/hibernate-reactive.adoc

```diff
@@ -357,27 +357,45 @@ public class SomeTest {
 ----
 ====
 
-[[hr-named-datasource]]
-==== Named data sources
+[[multiple-persistence-units]]
+=== Multiple persistence units
 
-Hibernate Reactive supports having named data sources
+==== Setting up multiple persistence units
+
+In a similar fashion to Hibernate ORM, Hibernate Reactive supports multiple persistence units.
+
+Users can define multiple persistence units and datasources, and they can mix blocking and reactive datasources.
```

Comment:
```suggestion
You can define multiple persistence units and datasources, and they can mix blocking and reactive datasources.
```

I'm sure there are guidelines encouraging this phrasing instead of "Users"... but I don't know where exactly. @sheilamjones would probably know? (Hey Sheila!)